.github/workflows/pullrequest.yml

```
@@ -15,6 +15,22 @@ jobs:
           distribution: 'temurin'
           java-version: 17
           cache: 'gradle'
+      - name: Check if PR user has API repo
```

```suggestion
      - name: Check if the user has forked the API repo
```
or `author` instead of `user`

---

https://github.com/GeyserMC/Geyser/pull/3535/commits/5ebfc0f361518d72b5c69c9f0b46899ff9b3e9ae#diff-ce9a2f9036cc2a5aaf1f78839c4f65c7a12808822e0dd0f3552471921ac5e007L18-R18